build.sbt

```diff
@@ -28,9 +28,9 @@ lazy val riddlIdeaPlugin: Project = Root(
     buildInfoPackage := "com.ossuminc.riddl.plugins.idea",
     buildInfoObject := "RiddlIDEAPluginBuildInfo",
     description := "The plugin for supporting RIDDL in IntelliJ",
-    libraryDependencies ++= Dep.testing ++ Dep.basic :+ Dep.fansi,
+    libraryDependencies ++= Dep.testing ++ Dep.basic,
     Test / parallelExecution := false,
-    scalaVersion := "3.4.2",
+    scalaVersion := "3.5.0",
```

Comment:
That might be a bit premature. In fact 3.4.2 is premature.  It also might mean incompatibility with 3.4.2 built riddl libraries? Any issues?